pretixprint/app/src/main/res/values/strings.xml

```diff
@@ -97,6 +97,7 @@
     <string name="btn_prev">PREVIOUS</string>
     <string name="protocol_fgl">FGL ticket printer (e.g. Boca, Practical Automation, …)</string>
     <string name="protocol_slcs">SLCS label printer (e.g. Bixolon, Metapace, …)</string>
+    <string name="protocol_brother">Brother label printer</string>
```

Comment:
Do we have any indication whether it only works for certain models or series, i.e. should we write "Brother label printer (QL series)"?

Reply:
added :)